lib/utilities/constants.dart

```diff
@@ -8,4 +8,5 @@ const pokedexTitle = 'Pokedex';
 const imageURL = 'https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/other/home/[index].png';
 const indexKey = '[index]';
 
-const errorMessageText = 'Something went wrong';
+const snackBarErrorMessage = 'Something went wrong!';

```

Comment:
errorMessage is enough for its name here, remember this message won't only be usable for snackbar it could be in any other case as well 🙂